Jenkinsfile

```diff
@@ -46,19 +46,34 @@ pipeline {
 		ANDROID_NDK = ""
 
 		PYTHONUNBUFFERED = "true"
+
 		JACOCO_XML = "Paintroid/build/reports/coverage/debug/report.xml"
+		JACOCO_UNIT_XML = "Paintroid/build/reports/jacoco/jacocoTestDebugUnitTestReport/jacocoTestDebugUnitTestReport.xml"
 
 		// place the cobertura xml relative to the source, so that the source can be found
-		COBERTURA_XML = "Paintroid/src/main/java/coverage.xml"
+		JAVA_SRC = "Paintroid/src/main/java"
+
 	}
 
 	options {
 		timeout(time: 2, unit: 'HOURS')
 		timestamps()
-                buildDiscarder(logRotator(numToKeepStr: '30'))
+		buildDiscarder(logRotator(numToKeepStr: '30'))
 	}
 
 	stages {
+		stage('Cleanup') {
+			steps {
+				// Clean all files and directories not tracked by git and that are not part of .gitignore.
+				// This includes the code coverage files that could otherwise lead to false positives
+				// during static analysis.
+				// Note: The build directory is not cleaned, this is done by gradle.
+				sh 'git clean -fd'
```

Comment:
Jenkins (resp the job) is configured to do a 'git clean -fdx' ('Clean before checkout'-option) on the used repository right before running the job. Therefore this stage 'duplicates' that task and would be a no-op.